src/app-state.ts

```diff
@@ -378,12 +380,20 @@ function useCartItemsState() {
     getCartItems(mcart).then(res => {
       setCartData(res.data.filter(({ type }) => type === 'cart_item' || type === 'custom_item'));
       setPromotionItems(res.data.filter(({ type }) => type === 'promotion_item'));
-      setCount(res.data.reduce((sum, { quantity }) => sum + quantity, 0));
-      setTotalPrice(res.meta.display_price.without_tax.formatted)
+      const itemQuantity = res.data.reduce((sum, { quantity }) => sum + quantity, 0);
+      setQuantity(itemQuantity - count);
+      setCount(itemQuantity);
+      setTotalPrice(res.meta.display_price.without_tax.formatted);
+      if (!showCartPopup && itemQuantity - count !== 0) {
+        setShowCartPopup(true);
+        setTimeout(() => {
+          setShowCartPopup(false);
+        }, 3200);
```

Comment:
Did we get the value for this delay from UX? Let's just verify with Bonnie if this is fast/slow enough